IncidentFields/incidentfields.json

```diff
@@ -1,4 +1,5 @@
 {
+	"releaseNotes": "nothing new",
```

Comment:
Don't write "nothing new". It appears in the release notes. 

If there is no change here. Revert to original file.

Reply:
Can't be reverted so I commited it with `"releaseNotes": "-"` as Ben suggested